libs/vm-modules/tests/unit/state_tests.cpp

```diff
@@ -403,14 +416,23 @@ TEST_F(StateTests, test_serialisation_of_structured_data)
       assert(arr_u64.count() == 1);
       assert(arr_u64[0] == 200u64);
 
+      return data.getBuffer("buffer");
     endfunction
   )";
 
   EXPECT_CALL(toolkit.observer(), Exists(state_name));
-  EXPECT_CALL(toolkit.observer(), Read(state_name, _, _));
+  EXPECT_CALL(toolkit.observer(), Read(state_name, _, _)).Times(AnyNumber());
```

Comment:
Presumably the number of times this is called is either 1 or 2. Might be better to stipulate this.

Reply:
Well, this is because of stroring binary data (e.g. String, Buffer, UInt256), where the length is dynamic (hence `AnyNumber()` usage). I can change it to specific number necessary for the current test (I believe it would be `2` in this particular test data configuration).

Reply:
The point I wanted to make with using the `AnyNumber()` was that this aspect is *not* relevant for this test objective (it has been already tested by many unit tests that `Read(...)` method works properly in terms of how many times it is necessary to be called when storing data).